<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secure Chat - Create Account</string>
  </property>
  <property name="windowOpacity">
   <double>0.990000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(54, 54, 54)</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:26pt; color:rgb(141, 141, 211)</string>
   </property>
   <property name="text">
    <string>Secure Chat</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16pt;color:rgb(202, 100, 151)</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>209</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(232, 232, 232);font-size:14pt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPassword">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>348</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(232, 232, 232);font-size:14pt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>339</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16pt;color:rgb(202, 100, 151)</string>
   </property>
   <property name="text">
    <string>Confirm Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signupButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>409</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(167, 167, 167)</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16pt;color:rgb(202, 100, 151)</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>279</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(232, 232, 232);font-size:14pt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16pt;color:rgb(222, 222, 222)</string>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
